README.md

```diff
@@ -5,32 +5,33 @@
 Convert from Tensorflow to CoreML
 =================================
 
-Python package to convert from Tensorflow to CoreML format. To get the latest
-version of `tfcoreml`, please run:
+[coremltools](https://github.com/apple/coremltools) (Recommended approach)
+--------------------
+
+
+For converting TensorFlow models to CoreML format, the recommended approach is to use `coremltools 4.0` python package.
+
+Please see the  coremltools documentation on [Tensorflow conversion](https://coremltools.readme.io/docs/tensorflow-conversion) for example usage.
+
+To get the latest version of coremltools:
 
 ```shell
-pip install --upgrade tfcoreml
+pip install coremltools==4.0b1
 ```
 
-For the latest changes please see the [release
-notes](https://github.com/tf-coreml/tf-coreml/releases).
+For the latest changes in coremltools, please see the [release notes](https://github.com/apple/coremltools/releases/).
 
-Usage
-------
+tfcoreml 
+---------
 
-Please see the Tensorflow conversion section in the [Neural network
-guide](https://github.com/apple/coremltools/blob/master/examples/NeuralNetworkGuide.md)
-on how to use the converter. 
+`tfcoreml` package is **no longer maintained**. It can be used only if the **primary** deployment target is `iOS 12` or earlier. To install `tfcoreml`, please run:
+
+```shell
```

Comment:
this can be changed to `pip install --upgrade tfcoreml`, 
in case bug fixes go in and the version 2.1, 2.2 etc are released

Reply:
Done